content/blogs/release-0-19.md

```diff
@@ -0,0 +1,769 @@
+
+
+### Managed Roles
+
+This release also adds **Managed Roles**, a set of read-only roles that are fully managed by Kestra. These roles—**`Admin`, `Editor`, `Launcher`, and `Viewer`**—are designed to simplify permission management, ensuring that users automatically receive the necessary permissions for new features without manual updates.
+
+**How Managed Roles Work?** Managed Roles cannot be edited or customized. When users attempt to add or remove permissions from these roles, a friendly error message will appear: _"Managed roles are read-only. Create a custom role if you need fine-grained permissions."_
+
+This ensures that the predefined roles remain consistent and aligned with any updates or new features introduced in Kestra.
+
+One of the key advantages of Managed Roles is that they stay up to date automatically. When Kestra adds new features, such as the **KV Store**, users with Managed Roles (like `Admins`) will automatically have the appropriate permissions to access these new capabilities. This removes the need for admins to manually update permissions for each new feature.
+
+If more granular control is needed, you can still create custom roles tailored to specific requirements. For most users, [Managed Roles](https://github.com/kestra-io/kestra-ee/issues/1327) provide a convenient, hands-off approach to role and permission management, ensuring access to all new features without any extra work.
+
+::alert{type="info"}
+Note that Managed Roles are not the same as [Default Roles](https://kestra.io/docs/configuration-guide/enterprise-edition#default-role-from-configuration). A default role is a role that will be assigned by default to every new user joining your instance which is useful for users automatically created via SSO. Managed Roles, on the other hand, are predefined roles that cannot be edited or customized. You can assign one of the Managed Roles as a Default Role for all new users. In this release, we've [enhanced](https://github.com/kestra-io/kestra-ee/issues/1344) the Default Role configuration to include an optional `tenantId` allowing you to restrict the default role access only to a specific tenant when needed (e.g. development, staging, production).
+::
+
+### New Permissions View
+
+The previous permissions dropdown system was tedious and time-consuming, requiring you to manually select each permission and its corresponding actions in order to configure a role.
+
+Kestra 0.19 [introduces](https://github.com/kestra-io/kestra-ee/issues/874) a new **view for permissions management** to simplify selecting the required permissions without having to manually click through every dropdown element. This new view allows you to check a parent element, like `FLOWS`, and automatically select all associated actions (`CREATE`, `READ`, `UPDATE`, `DELETE`). This eliminates tedious clicks needed to configure roles.
+
+![permissions_tree_view](/blogs/release-0-19/permissions_tree_view.png)
+
+---
+
+### Keeping You Logged In
```

Comment:
Have we considered a 'Notable bug fixes' section or alike in the past? This really sounds like it and I find that 'Keeping You Logged In' is not as impactful as other features we highlight (e.g. in the ToC)

Reply:
it might seem that way but in fact this has been **the #1 most requested feature across EE customers** -- it's not an exaggeration to say that EVERY customer reported this issue and we struggled to implement a solution for two releases and now it's finally here